### DISCUSSION TITLE
This idea is awesome ✨

### DISCUSSION THREAD

**Discussion by Gin-Quin** [Rezact/Rezact#2] [General]
Hey,

Just a message to say that I think you've got a stroke of genius. This is reactivity in its most simple form. Svelte used to be the simplest framework out there, but THIS is taking the crown.

I see you are using a Vite plugin to parse and transform the code to bring reactivity. It could be nice to explain your "magic" in the README, developers don't want to use something when they don't understand how it works.

Also, it could be worth to try to develop a Bun plugin. Bun is trending but there are few frameworks here that use all its possibilities during development. I'm thinking about [Bun's macros](https://bun.sh/blog/bun-macros), which are very powerful as well as unique in the ecosystem.

Starting a new JS framework is very hard, because people are used to see "this new branding JS frameworks that will change everything but changes nothing actually". See how Solid struggle, even though it's React but 10 times better.

Going with Bun as a target of choice could be a good strategy to find your place among other JS frameworks. Bun has plugins inspired by Vite (inspired by Rollup), so it should not be too complicated.

Anyway, good luck with this project, I'll come regularly watch how it's evolving!

**Comment by zachlankton** [comment]
@Gin-Quin Thank you so much for the encouragement!

Absolutely, there is a lot more docs and README to do for this project and I eventually want to include an "Advanced" / "Under the hood" section that goes into details about how it all works and how to pull the escape hatch if you ever need to work directly with the framework methods.

I've also been thinking quite a bit about Bun and how it could be a good thing for this project, but I've been so preoccupied with this that I have tried it yet.  Will definitely be doing this soon.

Thanks again!
